webpack.config.js

```diff
@@ -0,0 +1,46 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const path = require('path');
+const Htmlwebpackplugin = require('html-webpack-plugin');
+
+module.exports = {
+  entry: './src/index.jsx',
+  output: {
+    path: path.resolve(__dirname, './public'),
+    filename: 'bundle.js',
+    sourceMapFilename: 'bundle.map'
+  },
+  devServer: {
+    overlay: true,
+    stats: 'minimal',
+  },
+  resolve: {
+    alias: {
+      components: path.resolve(__dirname, './src/components/')
+    },
+    extensions: ['.js', '.jsx']
+  },
+  devtool: '#source-map',
+  module: {
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        exclude: /(node_modules)/,
+        use: ['babel-loader', 'eslint-loader']
+      },
+      {
+        test: /\.css$/,
+        use: ['style-loader', 'css-loader']
+      },
+      {
+        test: /\.(png|jpg|gif|jpeg)$/,
+        loader: 'file-loader',
+      }
+    ]
+  },
+  mode: 'development',
```

Comment:
If you're setting mode from the cli command, we don't need this.

Reply:
okay